feature-libs/organization/administration/styles/_list.scss

```diff
@@ -73,15 +73,36 @@
   }
 }
 
-%organizationTitle {
-  .title {
-    h3 {
-      button {
-        border: none;
-        background-color: inherit;
-        cx-icon {
-          pointer-events: none;
-          color: var(--cx-color, var(--cx-color-info));
+@include forVersion(6.8) {
```

Comment:
The only differences here that I see are that we add the same style to `h2` and include the type. Because I don't see any property changes or removals, I think we can add something like this to the bottom of the class.

```
@include featureStyles('flag') {
    %organizationTitle {
        .title {
            h2, h3 {
                @include type('3', 'normal');
            }
        }
    }
}
```